scss/sections/_informations.scss

```diff
@@ -1,51 +1,102 @@
 #informations {
-	display: flex;
-	width: 100%;
-	.informations_container {
-		display: flex;
-		flex-direction: column;
-		width: 100%;
-		max-width: 1400px;
-		margin: 0 auto 20px;
-		margin-bottom: 40px;
-		.information_block {
-			display: flex;
-			flex-direction: row;
-			.information_block_decription {
-				width: 50%;
-				color: black;
-				font-weight: normal;
-				padding: 2% 5% 5% 0%;
-				text-align: left;
-				h1 {
-					font-size: 40px;
-					color: black;
-					text-indent: 40px;
-					padding: 2% 0%;
-					margin-bottom: 15px;
-				}
-				h2 {
-					font-size: 28px;
-					margin-bottom: 20px;
-					padding-bottom: 1.5%;
-					text-indent: 40px;
-					padding-top: 10%;
-				}
-				p {
-					font-size: 18px;
-					padding-left: 10%;
-					line-height: 180%;
-				}
-			}
-			.information_image {
-				display: flex;
-				width: 50%;
-				img {
-					display: block;
-					width: 100%;
-					object-fit: contain;
-				}
-			}
-		}
-	}
+  display: flex;
+  width: 100%;
+
+  .informations_container {
+    display: flex;
+    width: 100%;
+    max-width: 1400px;
+    margin: 0 auto 40px;
+    flex-direction: column;
+
+    .information_block {
+      display: flex;
+      flex-direction: row;
+
+      .information_block_decription {
+        width: 50%;
+        padding: 2% 5% 5% 0%;
+        color: black;
+        font-weight: normal;
+        text-align: left;
+
+        h1 {
+          color: black;
+          padding: 2% 0%;
+          margin-bottom: 15px;
```

Comment:
but for fonts and elements which contains text its ok to have fixed dimensions in padding/margin if its necessary.
it's necessary here to have margin here instead default margin?